BetterMD/elements/table.py

```
@@ -14,92 +14,153 @@
-        return f"+-{"-+-".join([t*"-" for t in top])}-+\n{content}\n+={"=+=".join([t*"=" for t in top])}=+"
     
+        rows = [
+            [
+                e.to_rst().splitlines() for e in row.data
+            ] for row in symbol.data
+        ]
+
+        def op_get(list:'list', i:'int', d):
+            try:
+                return list[i]
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)